client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -12,11 +12,9 @@ import FormLabel from 'calypso/components/forms/form-label';
 import FormLegend from 'calypso/components/forms/form-legend';
 import FormRadio from 'calypso/components/forms/form-radio';
 import FormSectionHeading from 'calypso/components/forms/form-section-heading';
-import FormSelect from 'calypso/components/forms/form-select';
 import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
 import FormTextInput from 'calypso/components/forms/form-text-input';
 import FormTextarea from 'calypso/components/forms/form-textarea';
-import StateSelector from 'calypso/components/forms/us-state-selector';
```

Comment:
It might be a useful component for someone else to use? If it helps to reduce the build though, we can remove it. Would it be better to do it on a separate PR for visibility and to make it easier to revert later if someone wants the component?

Reply:
I agree it could be useful, but there's also a chance it may be unused forever 😉 

It's easy enough to dig out of history anyway. 

A separate PR to clean it up would be ace! Let me know if you need any help or a review.